src/ui/pages/Overview/BackupReminder.tsx

```diff
@@ -74,9 +82,18 @@ function BackupReminderComponent({ onDismiss }: { onDismiss: () => void }) {
             <UIText kind="body/accent">I{apostrophe}ll take the risk</UIText>
           </Button>
           <Button
-            onClick={handleHide}
+            onClick={(event) => {
+              handleHide();
+              if (notBackedUpWalletGroups.length === 1) {
+                openHrefInTabView(event);
+              }
+            }}
             as={UnstyledLink}
-            to="/settings"
+            to={
+              notBackedUpWalletGroups.length > 1
```

Comment:
May be it is a bit safer to swap condition parts. So we will not fall into the second line if length === 0

Reply:
yeah, I think its would be even better to check for `=== 1` ✅